src/main/resources/deeplink/v1/parsePlayUrl.js

```diff
@@ -27,6 +27,9 @@ function parseForPlayApp(scheme, hostname, pathname, queryParams, anchor) {
 		pathname = pathname.slice(0, -1)
 	}
 
+    hostnameCaseSensitive = hostname
+    pathnameCaseSensitive = pathname
```

Comment:
Instead of "case sensitive", I was thinking of `originalPathname`, like `originalUrl` used in the code. But fine.

Please remove `hostnameCaseSensitive` in both files, it's not used.

Reply:
Shame on me @Loic-Dumas : `originalPathName` 🙃🙈 "N"